server/handlers.go

```diff
@@ -68,6 +69,14 @@ func serveAddress(r *http.Request) string {
 		psa = r.URL.Host
 	}
 
+	psa = strings.TrimRight(psa, "/")
+
+	if config.Configuration.Server.BaseURL != "" {
```

Comment:
Let's make default BaseURL "/" so we don't need extra logic.

Let's add similar clauses to 67-69 for scheme.

Add a comment `// Preferred server host` & rename psa -> psh